### DISCUSSION TITLE
Keycloak 19.0.2(Server Info Missing) new admin suggestions

### DISCUSSION THREAD

**Discussion by razbag** [keycloak/keycloak#14688] [Archive]
Couple of suggestion for new admin console(theme)

Please keep those features those are really good to have in UI

**Server Uptime info** 
![image](https://user-images.githubusercontent.com/18428312/193405099-9c62007b-cab3-4d28-85ad-650c16857683.png)

**Memory** 
![image](https://user-images.githubusercontent.com/18428312/193405035-3818a48d-d441-41e8-96c8-83c358a53f88.png)

**System**
![image](https://user-images.githubusercontent.com/18428312/193405055-7a9ed943-435d-425d-b031-f776830597b4.png)

**Comment by hookenful** [comment]
Full agree, missing them

**Comment by jbman** [comment]
On master realm,the version is available.
![image](https://user-images.githubusercontent.com/789496/196957407-86c697a6-c658-45cc-b9a8-4b19c8c55d02.png)

I think it would be good to have at least this information available independently from the selected realm.

**Reply by razbag** [reply]
Yeah but only version is available I am really missing Memory info , System Settings  and Server Uptime :)

**Comment by jbman** [comment]
Created a concrete proposal how to include it in the Admin UI: https://github.com/keycloak/keycloak-ui/issues/3617
